### PR TITLE
remove arg0 file input and add validation

### PR DESCRIPTION
**Pull request by henry739**
re AB#9604

### CODE REVIEW

File: readeventsfile.go [hunk 1]
```diff
@@ -16,18 +15,7 @@ var (
...
-		return nil, fmt.Errorf("filename expected as first positional command argument")
-	}
-	return ReadVerifiableEventsFromFile(cCtx.Args().Get(0))
-}
-
-func readArgs0FileOrStdIoToVerifiableEvent(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
-	if cCtx.Args().Len() > 0 {
-		return ReadVerifiableEventsFromFile(cCtx.Args().Get(0))
-	}
+func stdinToVerifiableEvent(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
```
Comment (honourfish) [inline readeventsfile.go:18]: much better name, but can we have the plural? `stdinToVerifiableEvents`?

Same for decoded events variant

File: readeventsfile.go [hunk 2]
```diff
@@ -82,25 +53,16 @@ func VerifiableEventsFromData(data []byte) ([]logverification.VerifiableEvent, e
 		return nil, err
 	}
 
-	// TODO: validate the verifiable events are not empty
+	for _, event := range verifiableEvents {
```
Comment (honourfish) [inline readeventsfile.go:56]: sweet!

File: readeventsfile.go [hunk 3]
```diff
@@ -82,25 +53,16 @@ func VerifiableEventsFromData(data []byte) ([]logverification.VerifiableEvent, e
...
+		validationErr := event.Validate()
```
Comment (honourfish) [inline readeventsfile.go:57]: i like the naming of the err variable too!

**Review by robinbryce — APPROVED** [review]
LGTM

**Pull request merged**